forJIS7/src/main/java/JavaHomework/Lectures_17/Stream/ThirdTask/ProductService/ProductService.java

```diff
@@ -2,48 +2,13 @@
 
 import JavaHomework.Lectures_17.Stream.ThirdTask.Model.Product;
 
-import java.util.Comparator;
 import java.util.List;
-import java.util.stream.Collectors;
 
-import static JavaHomework.Lectures_17.Stream.ThirdTask.Repository.ProductRepository.*;
+public interface ProductService {
 
-public class ProductService implements Service {
+    List<Product> findProductByDiscount();
 
-    @Override
-    public List<Product> findProductByDiscount() {
-        return productList.stream()
-                .filter(it -> it.getDiscount() >= 30)
-                .collect(Collectors.toList());
-    }
+    Product maxProductPrice();
```

Comment:
This is the method. 
So you need to rename: findMaxProductPrice();
So you need to rename: findMinProductPrice();

Reply:
why rename them?